DHIS2/log_notificator/log_notificator.py

```diff
@@ -0,0 +1,66 @@
+import argparse
+import datetime
+import json
```

Comment:
Use:
```suggestion
from datetime import datetime
```
 to avoid the `datetime.datetime.*` code.